src/core/server/status/plugins_status.test.ts

```diff
@@ -231,6 +253,75 @@ describe('PluginStatusService', () => {
         { a: { level: ServiceStatusLevels.available, summary: 'a available' } },
       ]);
     });
+
+    it('updates when a plugin status observable emits', async () => {
```

Comment:
This test case was actually missing, even though it's the most likely behavior from a plugin.